pkg/minikube/detect/detect.go

```diff
@@ -140,12 +140,46 @@ func SocketVMNetInstalled() bool {
 	if runtime.GOOS != "darwin" {
 		return false
 	}
-	_, err := os.Stat(viper.GetString("socket-vmnet-path"))
+	return SocketVMNetPath() != "" && SocketVMNetClientPath() != ""
+}
+
+// SocketVMNetPath returns the path of socket_vmnet
+func SocketVMNetPath() string {
+	return checkSocketVMnetInstallLocations("socket-vmnet-path", "/var/run/socket_vmnet")
```

Comment:
is this correct ("socket-vmnet-path") ? or meant to be a var ?

Reply:
This is correct, it's a flag key, we check if the user set their own value, and if they do just use what they provided.

Reply:
sounds good